common/models/job-monitor.js

```diff
@@ -6,72 +6,22 @@ const _ = require('lodash'),
 
 module.exports = function(JobMonitor) {
 
```

Comment:
@lukewendling To my understanding, we did this because if we ever reran a job monitor it would reset the posts and get everything ready for a re-run...is this not the case anymore?